### ISSUE TITLE
When in fabric 1.19.2 with version 1.0.3 it disappears upon message sent

### ISSUE THREAD

**Issue by rasgajag** [apace100/show-me-what-you-got#9]

**Comment by similarly** [comment]
Same problem, here's my modlist, but I have no idea what it could be conflicting with. I think I have 1 mod that does minor changes to chat, I'll try removing it later. 
[modlist.txt](https://github.com/apace100/show-me-what-you-got/files/11354450/modlist.txt)

**Comment by ForionOG** [comment]
same issue have it installed on server and client but the mod wont sent the message at all

**Comment by apace100** [comment]
Could you provide logs from the server and the client? Did you make sure that both are using the same version of the mod?

**Comment by minc727** [comment]
This happens to me as well.  Both in single player and in multiplayer.  Both times, when I hit enter to send the message, I get the clientside error:

```
  <log4j:Event logger="net.minecraft.class_1255" timestamp="1686676972019" level="FATAL" thread="Render thread">
    <log4j:Message><![CDATA[Error executing task on Client]]></log4j:Message>
    <log4j:Throwable><![CDATA[java.lang.NullPointerException: Cannot invoke "net.minecraft.class_640.method_2966()" because "playerInfo" is null
	at net.minecraft.class_7594.handler$ebo000$collective$showMessageToPlayer(class_7594.java:2531)
	at net.minecraft.class_7594.method_44943(class_7594.java:193)
	at net.minecraft.class_7594.method_44768(class_7594.java:165)
	at net.minecraft.class_7594$1.accept(class_7594.java:123)
	at net.minecraft.class_7594.method_44818(class_7594.java:99)
	at net.minecraft.class_7594.method_44733(class_7594.java:113)
	at net.minecraft.class_634.method_43595(class_634.java:833)
	at net.minecraft.class_7438.method_43629(class_7438.java:27)
	at net.minecraft.class_7438.method_11054(class_7438.java:11)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:157)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:131)
	at net.minecraft.class_1255.method_5383(class_1255.java:116)
	at net.minecraft.class_310.method_1523(class_310.java:1129)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
]]></log4j:Throwable>
  </log4j:Event>
```

I don't see anything in the server logs (at least, in latest.log) or in the server console etc.  Both server and client are running v1.0.3

Attached are the modlist and the log for me joining a single player world and trying to ping an item.

[latest.log](https://github.com/apace100/show-me-what-you-got/files/11737628/latest.log)
[modlist.txt](https://github.com/apace100/show-me-what-you-got/files/11737629/modlist.txt)

The modpack I'm running is Prominence [FABRIC] V1.70, and the mod version I'm running is 1.0.3

**Comment by ForionOG** [comment]
Uuhm so yeah i had found the problem.... it had to do with the collective mod for as far as i know... once u use that mod with areas or whatever mod serilium made it will not work

**Comment by ForionOG** [comment]
Okay did some in-game tests Show me what you got and Areas are the only ones incompatible with each other.
Show me what u got won't let you sent you item in chat if u use the mod areas.
and not importand for this mod areas will not let you choose your own name for the area when using show me what u got

**Comment by SCPRedMage** [comment]
I'm getting this error on a fresh 1.19.2 profile with just the Fabric API, SMWYG, and Collective.

**Comment by SCPRedMage** [comment]
I've submitted an issue on the Collective mod's GitHub, with a fix that I've confirmed fixes the issue.

https://github.com/ricksouth/serilum-mc-mods/issues/1818
![image](https://github.com/apace100/show-me-what-you-got/assets/17621208/74145314-2fab-48d8-9a23-1f2d213aaf7d)

**Comment by SCPRedMage** [comment]
This has been fixed in Collective 6.59.
